### ISSUE TITLE
[$40][MSFT-242] [Registration] JAWS : Banner region is announced differently in Windows - Chrome

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2897]
**Steps to Reproduce**
1. Launch JAWS 
2. Launch "https://accounts.topcoder.com/member/registration" in Windows / Chrome
3. Navigate using R (for Regions)

**Expected Result**
It should be announced as "Banner"

**Actual Result**
Note that "Banner region" is announced as "vm dot register form form"

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 3. Understandable
**WCAG Level**: AAA
**WCAG Success Criterion**: 3.1.3 Unusual Words
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=1a9ec9e4%2D985a%2D449c%2D9e7a%2Dbb83cbe69ccb&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FArun%5FA11Y%2FJAWS%20Banner%20region%20announced%20differently%20in%20Chrome

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096234 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096234 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096234 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096234 has been updated - it has been assigned to afrisalyp.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2897 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2897 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[500]: Challenge for the updated issue 2897 is creating, rescheduling this event<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096750 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096750 has been updated - it has been assigned to afrisalyp.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by nkumar-topcoder** [comment]
Please #2866 & #2897 has same fix.

**Comment by drasticdpk** [comment]
Operating System: Windows
Browser: Chrome
URL : https://accounts.topcoder-dev.com/member/registration

**Comment by drasticdpk** [comment]
Verified on 
Operating System: Windows
Browser: Chrome

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30096750<br/><br/>```This is an automated message for crazyk via Topcoder X```
